packages/slice-machine/src/apiClient.ts

```diff
@@ -30,15 +31,31 @@ export const saveCustomType = (
   return axios.post("/api/custom-types/save", requestBody, defaultAxiosConfig);
 };
 
+/** Slice Routes **/
+
+export const createSlice = (
+  sliceName: string,
+  libName: string
+): Promise<{ variationId: string }> => {
+  const requestBody: SliceBody = {
+    sliceName,
+    from: libName,
+  };
+
+  return axios
+    .post(`/api/slices/create`, requestBody, defaultAxiosConfig)
+    .then((response: AxiosResponse<{ variationId: string }>) => response.data);
+};
+
 /** Auth Routes **/
 
 export const startAuth = (): Promise<AxiosResponse<Record<string, never>>> =>
   axios.post("/api/auth/start", {}, defaultAxiosConfig);
 
 export const checkAuthStatus = (): Promise<CheckAuthStatusResponse> =>
   axios
-    .post<CheckAuthStatusResponse>("/api/auth/status", {}, defaultAxiosConfig)
```

Comment:
Why change this?
the types are equivalent?

Reply:
I think that for using the same method for both createSlice and checkAuthStatus. I will check why I used this method, don't remember :)

Reply:
Typescript doesn't infers the type in the then if we put the type on the post generic